packages/fxa-auth-server/config/dev.json

```diff
@@ -129,7 +129,7 @@
           "product_metadata": {
             "webIconURL": "http://placekitten.com/512/512?image=11",
             "emailIconURL": "http://placekitten.com/512/512?image=5",
-            "downloadURL": "http://getfirefox.com",
+            "downloadURL": "http://127.0.0.1:8080/"
```

Comment:
Ack, I'd pulled this from master and it snuck its way into this commit, I'll see what I can do about it...